src/coreclr/scripts/superpmi_collect_setup.py

```diff
@@ -183,6 +183,7 @@
     "vcruntime140.dll",
     "vcruntime140_1.dll",
     "xunit.console.exe",
+    "ilc.exe"
```

Comment:
Nit: the list looks sorted, so can you place this after `ilasm`?

Reply:
Good catch. Done.